widgetsnbextension/src/extension.js

```diff
@@ -123,15 +123,12 @@ function register_events(Jupyter, events, outputarea) {
       Jupyter.notebook.kernel &&
       Jupyter.notebook.kernel.widget_manager;
     if (!manager) {
-      node.textContent =
-        'Error rendering Jupyter widget: missing widget manager';
-      return;
+      throw 'Error rendering Jupyter widget: missing widget manager';
```

Comment:
Note: In general in Javascript you'd want to throw an Error, and not a string directly. Was this intentional?

Reply:
Thanks for the review, I will update this